apps/google-analytics-4/frontend/src/hooks/useKeyService.tsx

```diff
@@ -1,33 +1,29 @@
+import React from 'react'
```

Comment:
This is a weird diff. Really this file just abstracts all the service keys and parameters into a single hook.

Reply:
love this improvement by the way. feel free to pull it out into a separate PR if that helps reduce the size of this one/is quick and easy